sql_joins/sql_joins.md

```diff
@@ -19,8 +36,23 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
-script: https://kit.fontawesome.com/83b2343bd4.js
+@pre_reqs
+
+Learners should have experience writing SQL code on single tables.  If you have successfully used a "SELECT... FROM... WHERE" SQL statement on a single table, and have at least seen "GROUP BY" commands in action, even if you would need help writing the GROUP BY code, you have enough code ability.  We also highly recommend that you understand the concepts of one-to-many data relationships and database normalization to get the most out of this module. 
+
+If you need to develop basic SQL fluency we recommend our module [SQL Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_basics/sql_basics.md).  For more intermediate topics, we suggest our module [SQL Intermediate](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_intermediate/sql_intermediate.md).  Finally, to learn about one-to-many data relationships and database normalization, consider our [Database Normalization](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/database_normalization/database_normalization.md) module.
+
+@end
+
+@version_history
+
+Previous Versions:
+
+* [1.0.1](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/d428e9f66a2161e96ea4ca32b42049fab2d27088/sql_joins/sql_joins.md#1): Original version, with improved feedback link
+
+@end
+
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
 
 script: https://cdn.jsdelivr.net/npm/alasql@0.6.5/dist/alasql.min.js
```

Comment:
I know that this has a different dataset from the other sql modules, does that mean that it shouldn't import the script from macros_sql.md? Or does it need the script itself right here to build the different database?

I think that, if my understanding is correct, the small database used in this module could also be included in the macro file as a second database with its own name so that other, future modules could potentially also use it easily. But I'm not totally sure how the AlaSQL code is building these databases so it might not make sense to do this.

Reply:
The issue is that the macros_sql.md pulls in a TON of unneeded data... I'd like to avoid that if possible.

Reply:
Ok got it! That makes sense.